src/bin/cql-stress-cassandra-stress/settings/command/user.rs

```diff
@@ -54,6 +54,7 @@ impl QueryDefinitionYaml {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
 pub struct QueryDefinition {
     pub cql: String,
     pub consistency: Option<Consistency>,
```

Comment:
If deriving `PartialEq`, `Eq` should be derived as well unless there is a reason not to.